### PR TITLE
Helm chart improvements

### PR DESCRIPTION
**Pull request by mmolnar**

### CODE REVIEW

**Comment by cregev** [comment]
@mmolnar, very nice! I did not there is a Helm Chart release.

Let me know when to review it... I went over most of the changes already.

**Comment by mmolnar** [comment]
I had to redo it as the original github action assumed there are multiple helm charts under `charts` directory and did not allow using a single chart in root of the repository.

I modified the original github action from Helm, striped it of unnecessary things (for us) and modified it to work for a single chart.
Currently it will make a new release for the chart after each change of `version` field inside `Chart.yaml`, tag the repository as `vx.x.x` where `x.x.x` is the version from `Chart.yaml` and update github pages to be avare of the new release.

Seems to work as can be seen in my fork:
https://github.com/mmolnar/statusbay-helm

I can even use the github pages directly as an helm repo:
```bash
helm template my-release --repo https://mmolnar.github.io/statusbay-helm/ statusbay-helm
```

File: .github/actions/release-chart-action/cr.sh [hunk 1]
```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env bash
+
+# Copyright The Helm Authors
```
Comment (mmolnar) [inline .github/actions/release-chart-action/cr.sh:3]: I am not sure how to indicate here that the original was authored by somebody else and it was modified.
Also, how do we do opensource licensing in Similarweb?

**Review by rs324 — APPROVED** [review]
Nice job.

**Pull request merged**